src/command/stamp/buy.ts

```diff
@@ -62,28 +62,39 @@ export class Buy extends StampCommand implements LeafCommand {
       this.waitUsable = await this.console.confirm('Would you like to enable it now?')
     }
 
-    const spinner = createSpinner('Buying postage stamp. This may take a while.')
+    const _estimated = BigNumber(Number(this.amount)).multipliedBy(BigNumber(2).pow(this.depth))
+    const PLURConvertionRate = BigNumber(10).pow(16)
+    const estimated = BigNumber(_estimated).dividedBy(PLURConvertionRate)
```

Comment:
The duplication of `estimated` and `_estimated` is not necessary here.

I suggest first creating `const PLURConvertionRate = BigNumber(10).pow(16)` and then you can compute `estimated` in one go. One side note is that `estimated` does not tell you what it estimates, so I think `estimatedCost` would be a better name.

Careful with the `Number(this.amount)`, since `this.amount` is already a `bigint`, converting it back to `Number` will make it lose precision.

So I think it could look like this:

```
const estimatedCost = BigNumber(this.amount).multipliedBy(BigNumber(2).pow(this.depth)).dividedBy(PLURConvertionRate) // in bzz
```

Reply:
The problem with this `BigNumber(this.amount)` is that this.amount is a bigint, and is not being accepted by BigNumber as a valid parameter type, that's why I cast it to `number`
![image](https://user-images.githubusercontent.com/1278025/186476432-29cf9968-a657-46c8-b569-60ff3edbc087.png)

Reply:
In that case it would be better to convert `this.amount` to string so that it does not lose precision.